specification/ingest/simulate/types.ts

```diff
@@ -27,10 +27,20 @@ import { ErrorCause } from '@_types/Errors'
 import { DateTime } from '@_types/Time'
 
 export class Ingest {
+  _redact?: Redact
```

Comment:
Should we add an availability annotation here as well? It probably makes no sense to have an empty class in pre 8.16 versions e.g. in case this gets backported.

Reply:
wait how could this get backported to 8.15?

Reply:
In any case it does make sense to add the annotation at this level I think?